package.json

```diff
@@ -35,9 +36,11 @@
     "gulp-nodemon": "^2.5.0",
     "gulp-sass": "^4.0.1",
     "gulp-sourcemaps": "^2.6.0",
+    "hmpps_offender_assessment_api": "file:assessments-api-client",
```

Comment:
_not a blocker_
Looking at the [npm docs](https://docs.npmjs.com/files/package.json#local-paths), this may need a `./` prefix to be explicit about the location (`file:./assessments-api-client`). But I'm not a JS ecosystem person 🤷‍♂️  It certainly worked for me locally.

Reply:
Weird, since I didn't make this change manually - it's the result of `npm install --save ./assessments-api-client`…

I'll keep it as it is instead of changing it by hand.